porter/v1/prometheus_alerts.proto

```diff
@@ -19,7 +21,11 @@ message Alert {
   string name = 1;
   string namespace = 2;
   InvolvedObjectType type = 3;
-  int32 desired_replicas = 4;
-  int32 available_replicas = 5;
-  int32 max_unavailable = 6;
+  // severity should be a string specifying how severe the alert is
+  string severity = 4;
+  // start_time holds the timestamp for when this status began
+  google.protobuf.Timestamp start_time = 5;
```

Comment:
Are we storing these anywhere? Changing the datatype at a given index will have unintended consequences and should be heavily avoided if possible. Consider using "deprecated" instead

Reply:
These are not really affecting clients right now, we should be fine. I am also creating a new table given the big number of changes.